frontend-opal/src/components/WatchList/SingleInstrument/SingleInstrument.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import InstrumentRow from 'components/WatchList/InstrumentRow/InstrumentRow';
+import ChartDetail from 'components/WatchList/ChartDetail/ChartDetail';
+import presetProps from 'components/WatchList/presetProps';
+import actions from 'actions/watchlists';
+import styles from './styles';
+
+function SingleInstrument({ data, index, toggleOneInstrument }) {
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity onPress={() => toggleOneInstrument(index, data.symbol)}>
```

Comment:
@hondahmd probably this is going to be conflicting with the button handling, so the 2 icons on the right will act as buttons, and you wont be able to click on them